f5_openstack_agent/lbaasv2/drivers/bigip/network_service.py

```diff
@@ -287,9 +287,12 @@ def assign_route_domain(self, tenant_id, network, subnet):
             network['route_domain_id'] = 0
             return
 
-        LOG.debug("Assign route domain get from cache %s" % network)
+        LOG.debug("\nAssign route domain get from cache %s."
+                  "\nThe network is %s." % (self.rds_cache, network))
```

Comment:
We should split log lines into 2, instead of add'\n'. It's not benefit for log search.